core/src/main/java/org/jivesoftware/sparkimpl/profile/AvatarPanel.java

```diff
@@ -291,7 +293,49 @@ public void setParentDialog(Dialog dialog) {
         this.dlg = dialog;
     }
 
+    /*
+     * Crop the given image file to the target dimensions 
+     * without changing aspect ratio.
+     * Save the resultant file into disk and return a reference
+     * to it.  
+     */
+    private File cropImageFile(File selectedFile){
+
+        int targetImageWidth = 100;
+        int targetImageHeight = 100;
+        File imageFile = null;
+
+        try {
+            BufferedImage avatarImageBuffered = ImageIO.read(selectedFile);
+            BufferedImage croppedImage;
+
+            int currentImageWidth = avatarImageBuffered.getWidth(null);
+            int currentImageHeight = avatarImageBuffered.getHeight(null);
+            
+            if(currentImageHeight == targetImageHeight && currentImageWidth == targetImageHeight){
+            	//no need to crop
+                croppedImage = avatarImageBuffered;
+                imageFile = selectedFile;
+            }else{
```

Comment:
I know that is this a petty pet peeve of mine, but could you please add some whitespace before and after curly braces? Instead of ``}else{`` please do ``} else {``. My life is incomplete without them. ;-)